db/migrate/20170531094819_move_conferences_to_organizations.rb

```diff
@@ -0,0 +1,21 @@
+class MoveConferencesToOrganizations < ActiveRecord::Migration
+  class TempConference < ActiveRecord::Base
+    self.table_name = 'conferences'
+  end
+
+  class TempOrganization < ActiveRecord::Base
+    self.table_name = 'organizations'
+  end
+
+  def change
+    add_reference :conferences, :organization, index: true
+    add_foreign_key :conferences, :organizations, dependent: :delete
```

Comment:
If the `conferences` table already has some conferences with NULL `organization_id`, will `add_foreign_key` work?